RecoTracker/TkTrackingRegions/plugins/PointSeededTrackingRegionsProducer.h

```diff
@@ -201,17 +198,13 @@ class  PointSeededTrackingRegionsProducer : public TrackingRegionProducer
 
     // create tracking regions (maximum MaxNRegions of them) in directions of the
     // points of interest
-    size_t n_points = points.size();
-    double eta_input, phi_input;
+    size_t n_points = etaPoints.size();
     int n_regions = 0;
     for(size_t i = 0; i < n_points && n_regions < m_maxNRegions; ++i ) {
 
-      eta_input          = points[i].getParameter<double>("eta");
-      phi_input          = points[i].getParameter<double>("phi");
- 
-      double x = TMath::Cos(phi_input);
-      double y = TMath::Sin(phi_input);
-      double theta = 2*TMath::ATan(TMath::Exp(-eta_input));
+      double x = TMath::Cos(phiPoints[i]);
+      double y = TMath::Sin(etaPoints[i]);
+      double theta = 2*TMath::ATan(TMath::Exp(-etaPoints[i]));
```

Comment:
@JanFSchulte can you replace these with `std::cos`, `std::sin`, `std::atan`, `std::exp` ?

Unless we have a central definition for mapping `eta` to `theta` ?